packages/material-ui/src/ObjectFieldTemplate/ObjectFieldTemplate.tsx

```diff
@@ -49,6 +57,17 @@ const ObjectFieldTemplate = ({
             {element.content}
           </Grid>
         ))}
+        {canExpand(schema, uiSchema, formData) && (
+          <Grid container justify="flex-end">
+            <Grid item={true}>
+              <AddButton
+                className="object-property-expand"
+                onClick={onAddClick(schema)}
+                disabled={false}
```

Comment:
This should be disabled if prop is disabled `disabled={disabled || readonly}`

Reply:
:facepalm: Will fix.